pkg/webservice/handlers_test.go

```diff
@@ -1544,7 +1595,7 @@ func assertParamsMissing(t *testing.T, resp *MockResponseWriter) {
 	assert.Equal(t, errInfo.StatusCode, http.StatusBadRequest)
 }
 
-func assertPartitionExists(t *testing.T, resp *MockResponseWriter) {
+func assertPartitionNotExists(t *testing.T, resp *MockResponseWriter) {
```

Comment:
there are other similar issues. for example: `assertQueueExists`, `assertApplicationExists`, and `assertNodeIDExists`. Maybe we can fix them at all?